__tests__/unit/flex/flex.test.js

```diff
@@ -94,6 +94,64 @@ describe('Test processBeforeValidate and processAfterValidate invocations', () =
     expect(action.processAfterValidate.mock.calls.length).toBe(2)
   })
 
+  test('Expect pass action to be triggered on empty validator ', async () => {
+    const config = `
+    version: 2
+    mergeable:
+      - when: pull_request.opened
+        name: "Greet a contributor"
+        validate: []
+        pass:
+          - do: comment
+            payload:
+              body: >
+                Thanks for creating a pull request! A maintainer will review your changes shortly. Please don't be discouraged if it takes a while.`
+
+    Helper.mockConfigWithContext(context, config)
+    let registry = { validators: new Map(), actions: new Map() }
+    const commentAction = new Action()
```

Comment:
The change in question where the logic to test the empty array are in each individual action that inherits `Action` -- for that reason, I don't feel this actually is testing the conditions. 

Would be good if the tests are validating the line in question:
`results && results.validationSuites && results.validationSuites.length && results.validationSuites[0].schedule`

Thoughts?

Reply:
@jusx fixed :)